model/src/main/kotlin/OrtResult.kt

```diff
@@ -246,18 +246,24 @@ data class OrtResult(
     /**
      * Return a map of all de-duplicated [Issue]s associated by [Identifier]. If [omitExcluded] is set to true, excluded
      * issues are omitted from the result. If [omitResolved] is set to true, resolved issues are omitted from the
-     * result.
+     * result. Issues with [severity][Issue.severity] below [minSeverity] are omitted from the result.
      */
     @JsonIgnore
-    fun getIssues(omitExcluded: Boolean = false, omitResolved: Boolean = false): Map<Identifier, Set<Issue>> {
+    fun getIssues(
+        omitExcluded: Boolean = false,
+        omitResolved: Boolean = false,
+        minSeverity: Severity = Severity.entries.min()
+    ): Map<Identifier, Set<Issue>> {
         val analyzerIssues = analyzer?.result?.getAllIssues().orEmpty()
         val scannerIssues = scanner?.getAllIssues().orEmpty()
         val advisorIssues = advisor?.results?.getIssues().orEmpty()
 
         val allIssues = analyzerIssues.zipWithCollections(scannerIssues).zipWithCollections(advisorIssues)
 
         return allIssues.mapValues { (_, issues) ->
-            issues.filterTo(mutableSetOf()) { !omitResolved || !isResolved(it) }
+            issues.filterTo(mutableSetOf()) {
+                (!omitResolved || !isResolved(it)) && it.severity >= minSeverity
+            }
```

Comment:
All in all, this could now look something like this:

        return allIssues.mapNotNull { (id, issues) ->
            if (omitExcluded && isExcluded(id)) return@mapNotNull null

            val filteredIssues = issues.filterTo(mutableSetOf()) {
                (!omitResolved || !isResolved(it)) && it.severity >= minSeverity
            }

            filteredIssues.takeUnless { it.isEmpty() }?.let { id to it }
        }.toMap()